src/server/server.ts

```diff
@@ -44,22 +43,25 @@ export async function getServer(): Promise<Express> {
   configureViews(server);
 
   // initialize components
-  await initAuth(server);
-  await initLists(server);
-  await initCookies(server);
-  await initSitemap(server);
-  await initFeedback(server);
-  await initDashboard(server);
-  await initDevelopment(server);
-  await initHealthCheck(server);
-
-  // error handlers
-  configureErrorHandlers(server);
 
+  try {
```

Comment:
@jenbutongit , unclear what this relates to.  Suggesting to wrap just the call to `configureErrorHandlers(server);`?  If so I don't think this is needed as errors should bubble up to prevent starting up in getServer()